geode-core/build.gradle

```diff
@@ -420,4 +420,10 @@ distributedTest {
   exclude "**/*\$*.class"
 }
 
+rootProject.combineReports.dependsOn(':geode-old-versions:test')
```

Comment:
I'll try and work out how to mitigate that task. Give me a day :)

Reply:
Sure, no rush... I wasn't trying to create work / re-prioritize work for anyone by making this PR, it was more intended as a "in case this is helpful... this might be a starting point on getting to Gradle 7..."

Reply:
@ryangardner I tested your PR on my side, and we can remove the line 423 addition above. It isn't part of the warning, and `geode-old-versions:test` has no real actions so it resolves to a no-op statement anyway.